.github/workflows/autofix.yml

```
@@ -1,15 +1,31 @@
-permissions: { contents: read }
+
+# `pull_request` already covers PR branches. Without the `push` branch filter,
+# every push to a PR branch ran this workflow twice for the same commit, the two
+# runs racing to push a fixup commit to the same branch.
+on:
+  push: { branches: [main] }
+  pull_request: {}
+
+permissions: {}
+
+concurrency: { group: autofix-$, { { github.ref } }, cancel-in-progress: true }
```

actionlint (1.7.12)

[error] expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)